internal/uvm/create_lcow.go

```diff
@@ -504,14 +513,31 @@ func makeLCOWVMGSDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_
 
 	// Point at the file that contains the linux kernel and initrd images.
 	doc.VirtualMachine.GuestState = &hcsschema.GuestState{
-		GuestStateFilePath:  vmgsFile.Name(),
+		GuestStateFilePath:  vmgsFileFullPath,
 		GuestStateFileType:  "FileMode",
 		ForceTransientState: true, // tell HCS that this is just the source of the images, not ongoing state
 	}
 
 	return doc, nil
 }
 
+func createCopy(dst, src string) error {
```

Comment:
probably could just use `"github.com/Microsoft/hcsshim/internal/copyfile".CopyFile` (or `"github.com/Microsoft/hcsshim/internal/winapi".CopyFileW` directly) instead of needing to open both.